src/pages/AdminPage/User/DriverInfo/DriverDetail/index.jsx

```diff
@@ -118,7 +142,7 @@ function DriverDetail() {
       bank: driverInfo.bank,
       holidays: driverInfo.holidays,
       introduction: driverInfo.introduction,
-      phoneNumber: driverInfo.phoneNumber,
+      phoneNumber: newPhoneNum,
```

Comment:
```suggestion
      phoneNumber: driverInfo.phoneNumber,
```
수정된 전화번호가 driverInfo에 있으니까, 이렇게 전송하는 게 맞는 것 같아
그리고 전화번호를 전송할 때는 '-' 문자를 제거하고 보내주면 좋겠어
010-0000-0000으로 표시하고 있는데, 이 값이 그대로 driverInfo에 들어가니까 api 호출할 때는 이 '-'문자를 제거하고 숫자만 보내도록!

Reply:
응 두 개 다 확인했어!
api 호출시 '-' 문자 제거까지 완료했어-!